framework/src/actions/CreateExecutionerAction.C

```diff
@@ -137,11 +134,17 @@ CreateExecutionerAction::act()
     _problem->getNonlinearSystem()._compute_initial_residual_before_preset_bcs = getParam<bool>("compute_initial_residual_before_preset_bcs");
   }
 
+  Moose::setup_perf_log.push("Create Executioner","Setup");
+  _moose_object_pars.set<FEProblem *>("_fe_problem") = _problem.get();
+  MooseSharedPointer<Executioner> executioner = MooseSharedNamespace::static_pointer_cast<Executioner>(_factory.create(_type, "Executioner", _moose_object_pars));
+  Moose::setup_perf_log.pop("Create Executioner","Setup");
+
```

Comment:
Why did these lines get moved? This isn't equivalent logic now, it means we have to have a problem in order to create an Executioner?